src/ncp/ncp_base.cpp

```diff
@@ -3069,12 +3135,14 @@ ThreadError NcpBase::SetPropertyHandler_MAC_PROMISCUOUS_MODE(uint8_t header, spi
         {
         case SPINEL_MAC_PROMISCUOUS_MODE_OFF:
             otPlatRadioSetPromiscuous(mInstance, false);
+            otSetLinkPcapCallback(mInstance, NULL, NULL);
```

Comment:
I don't think we should allow  `SPINEL_PROP_MAC_PROMISCUOUS_MODE` while the Thread interface is UP, and instead, rely on just on getting callbacks when directly bound to the radio.